$NetBSD: patch-ab,v 1.1 2010/06/15 08:26:54 wiz Exp $

Fix known incompatibility introduced in a recent version of OCaml.
The following patch, taken from
http://old.nabble.com/Bug-569260:-FTBFS:-expression-was-expected-of-type-string--%3E-string--%3E-string-td27540744.html ,
fixes it.

--- ptests/ptests.ml.orig	2009-08-31 15:38:55.000000000 +0000
+++ ptests/ptests.ml
@@ -54,7 +54,7 @@ module Filename = struct
       fun a b -> let r = temp_file a b in
 	cygpath r
     else
-      temp_file
+      (fun x y -> temp_file x y)
 end
 
 let default_env var value =
